benchmark/paginate/google.com-maps-search-paginate.bench.js

```diff
@@ -0,0 +1,28 @@
+import { fox } from '../../src/index.js';
+import { itRunMatrix } from '../lib/index.js';
+import { standardMatrix } from '../lib/matrix.js';
+import { checkIncreasingSize } from '../lib/checks.js';
+
+describe('paginate google.com maps restaurants search', async function() {
+  const matrix = standardMatrix(
+    {
+      fetcher: [['playwright', { headless: false }]],
```

Comment:
I know the other benchmarks have this, but actually it can just be

```
const matrix = standardMatrix();
```

we don't vary the fetcher anymore